### PR TITLE
Fix/tg 290

### PR DESCRIPTION
**Pull request by SergiiKudriashov**
Button mooved, servo renamed to switcher + fix of header layout for statistics

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/189" target="_blank">ReviewNinja

File: src/app/statistic/sensor-statistic/sensor-statistic.template.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="chart-holder">
-    <sh-servo-statistic-column-chart *ngIf="sensorId === 'servo'"
+    <sh-switcher-statistic-column-chart *ngIf="sensorId === 'switcher'"
         [deviceStatistic]="deviceStatistic">
-    </sh-servo-statistic-column-chart>
-    <sh-chart-widget *ngIf="sensorId !== 'servo'"
+    </sh-switcher-statistic-column-chart>
+    <sh-chart-widget *ngIf="sensorId !== 'switcher'"
```
Comment (borys-ihnatyev) [inline src/app/statistic/sensor-statistic/sensor-statistic.template.html]: move this `if` expression in the component controller under the function/property named `isChartWidgetVisible`

File: src/app/statistic/sensor-statistic/sensor-statistic.template.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <div class="chart-holder">
-    <sh-servo-statistic-column-chart *ngIf="sensorId === 'servo'"
+    <sh-switcher-statistic-column-chart *ngIf="sensorId === 'switcher'"
```
Comment (borys-ihnatyev) [inline src/app/statistic/sensor-statistic/sensor-statistic.template.html]: move this if expression in the component controller under the function/property named `isSwitcherWidgetVisible`

Comment (SergiiKudriashov) [reply src/app/statistic/sensor-statistic/sensor-statistic.template.html]: In two cases *ngIf="sensorId === 'switcher'". Maybe it is better to have one function
isVisible or is isChartOrSwitcherVisible?

**Pull request merged**